docs/topics/logs.md

```diff
@@ -0,0 +1,36 @@
+# Logs
+
+(logs)=
+
+We have various types of logs with different purposes in AMO. The following
+tables summarize their characteristics:
+
+
+| | `ActivityLog` / `LogEntry` |
```

Comment:
Did you consider using a markdown table or at least a multi column table? Seems the rows are repeated for each type and wondered if it's more readable as a single table? I downloaded the built docs site and it's unclear. 

Up to you.

Reply:
Yes, I tried that, that's actually how I originally had it, but it overflowed (the theme we're using only allows a `max-width` of `800px` on the content) so I figured I would make separate tables for now.

Reply:
Thought that might be the case 👍